### PR TITLE
[TEST] Add test case for tizen sensor

### PR DESCRIPTION
**Pull request by gichan-jang**
Add test case for tizen sensor

Signed-off-by: gichan-jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/tizen_capi/unittest_tizen_sensor.cc [hunk 1]
```diff
@@ -410,6 +410,100 @@ TEST (tizensensor_as_source, virtual_sensor_flow_05_n)
 
   g_free (pipeline);
 }
+
+/**
+ * @brief Test for tizen sensor get property
+ */
+TEST (tizensensor_as_source, get_propery_1)
```
Comment (anyj0527) [inline tests/tizen_capi/unittest_tizen_sensor.cc:417]: Minor typo: How about `get_propery_X` -> `get_property_X`  ?

File: tests/tizen_capi/unittest_tizen_sensor.cc [hunk 2]
```diff
@@ -410,6 +410,100 @@ TEST (tizensensor_as_source, virtual_sensor_flow_05_n)
...
+    g_object_set (sensor_handle, "mode", 0, NULL);
+    g_object_get (sensor_handle, "mode", &mode, NULL);
+    EXPECT_EQ (mode, 0);
+
+    g_object_set (sensor_handle, "framerate", 10, 1, NULL);
+    g_object_get (sensor_handle, "framerate", &freq_n, &freq_p, NULL);
+    EXPECT_EQ (freq_n, 10);
+    EXPECT_EQ (freq_p, 1);
+
+    status = 0;
```
Comment (anyj0527) [inline tests/tizen_capi/unittest_tizen_sensor.cc:457]: I think this line is unnecessary

File: tests/tizen_capi/unittest_tizen_sensor.cc (hunk 1 above)
Comment (gichan-jang) [reply tests/tizen_capi/unittest_tizen_sensor.cc]: Thanks! fixed.

File: tests/tizen_capi/unittest_tizen_sensor.cc (hunk 2 above)
Comment (gichan-jang) [reply tests/tizen_capi/unittest_tizen_sensor.cc]: Thanks! fixed.

**Pull request merged**